### PR TITLE
couple tweaks to the new how-we-structure projects

### PR DESCRIPTION
**Pull request by patkearns10**
#### TODO Link fail
Screen Shot 2022-06-02 at 2 18 46 PM

#### Link fail
Screen Shot 2022-06-02 at 2 15 26 PM

#### weird double plural
Also, we referenced entities as entity before (?)
Screen Shot 2022-06-02 at 2 19 52 PM

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](../contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](../website/docs/docs/guides/migration-guide)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/guides/best-practices/how-we-structure/2-staging.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ renamed as (
         status,
 
         -- numerics
-    amount as amount_cents,
+        amount as amount_cents,
```
Comment (gwenwindflower) [inline website/docs/guides/best-practices/how-we-structure/2-staging.md]: we fixed all this already so you'll need to repull and remove this commit

File: website/docs/guides/best-practices/how-we-structure/2-staging.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ select * from renamed
...
   - ✅ **Categorizing** (using conditional logic to group values into buckets or booleans, such as in the `case when` statements above)
-  - ❌ **Joins** — the goal of staging models is to clean and prepare individual source conformed concepts for downstream usage. We're creating the most useful version of a source system table, which we can use as a new modular component for our project. In our experience, joins are almost always a bad idea here — they create immediate duplicated computation and confusing relationships that ripple downstream — there are occasionally exceptions though (see [base models](TODO: link) below).
+  - ❌ **Joins** — the goal of staging models is to clean and prepare individual source conformed concepts for downstream usage. We're creating the most useful version of a source system table, which we can use as a new modular component for our project. In our experience, joins are almost always a bad idea here — they create immediate duplicated computation and confusing relationships that ripple downstream — there are occasionally exceptions though (see [base models](guides/best-practices/how-we-structure/2-staging#staging-other-considerations) below).
```
Comment (gwenwindflower) [inline website/docs/guides/best-practices/how-we-structure/2-staging.md:102]: nice catch!

File: website/docs/guides/best-practices/how-we-structure/5-the-rest-of-the-project.md [hunk 3]
```diff
@@ -93,7 +93,7 @@ We’ve focused heavily thus far on the primary area of action in our dbt projec
...
..., writing singular tests has become less and less necessary. It's extremely useful for work-shopping test logic, but more often than not you'll find yourself either migrating that logic into your own custom generic tests or discovering a pre-built test that meets your needs from the ever-expanding universe of dbt packages (between the extra tests in [`dbt-utils`](https://github.com/dbt-labs/dbt-utils) and [`dbt-expectations`](https://github.com/calogica/dbt-expectations) almost any situation is covered). One area where singular tests still shine though is flexibly testing things that require a variety of specific models. If you're familiar with the difference between [unit tests](https://en.wikipedia.org/wiki/Unit_testing) [and](https://www.testim.io/blog/unit-test-vs-integration-test/) [integration](https://www.codecademy.com/resources/blog/what-is-integration-testing/) [tests](https://en.wikipedia.org/wiki/Integration_testing) in software engineering, you can think of generic and singular tests in a similar way. If you need to tests the results of how several specific models interact or relate to each other, a singular test will likely be the quickest way to nail down your logic.
```
Comment (gwenwindflower) [inline website/docs/guides/best-practices/how-we-structure/5-the-rest-of-the-project.md:96]: 👍🏻

File: website/docs/guides/best-practices/how-we-structure/3-intermediate.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ models/intermediate
...
...strictly how to name them. The best guiding principle is to think about *verbs* (e.g. `pivoted`, `aggregated_to_user`, `joined`, `fanned_out_by_quanity`, `funnel_created`, etc.)  in the intermediate layer. In our example project, we use an intermediate model to pivot payments out to the order grain, so we name our model `int_payments_pivoted_to_orders`. It’s easy for anybody to quickly understand what’s happening in that model, even if they don’t know SQL. That clarity is worth the long file name. It’s important to note that we’ve dropped the double underscores at this layer. In moving towards business conformed concepts, we no longer need to separate a system and an entity and simply reference the unified entity if possible. In cases where you need intermediate models to operate at the source system level (e.g. `int_shopify__orders_summed`, `int_core__orders_summed` which you would later union), you’d preserve the double underscores. Some people like to separate the entity and verbs with double underscores as well. That’s a matter of preference, but in our experience there are often intrinsic connection between entities and verbs in this layer that make that difficult to maintain.
```
Comment (gwenwindflower) [inline website/docs/guides/best-practices/how-we-structure/3-intermediate.md:22]: 💯

File: website/docs/guides/best-practices/how-we-structure/2-staging.md [hunk 5]
```diff
@@ -216,5 +216,5 @@ This is a welcome change for many of us who have become used to applying the sam
...
+This guide follows the order of the DAG, so we can get a holistic picture of how these three primary layers build on each other towards fueling impactful data products. It’s important to note though that developing models does not typically move linearly through the DAG. Most commonly, we should start by mocking out a logical data model so we know we’re aligned with our stakeholders on output goals. Then, we’ll want to write the SQL to generate that output, and identify what tables are involved. Once we have our logic and dependencies, we’ll make sure we’ve staged all the necessary atomic pieces into the project, then bring them together based on the logic we wrote to generate our mart. Finally, with a functioning model flowing in dbt, we can start refactoring and optimizing that mart. By splitting the logic up and moving parts back upstream into intermediate models, we ensure all of our models are clean and readable, the story of our DAG is clear, and we have more surface area to apply thorough testing.
```
Comment (gwenwindflower) [inline website/docs/guides/best-practices/how-we-structure/2-staging.md]: i want to keep this actually. i'm specifically pushing people to sketch the output _before_ they write SQL as a best practice, and a spreadsheet is absolutely the best way to do that.

**Review by gwenwindflower — CHANGES_REQUESTED** [review]

File: website/docs/guides/best-practices/how-we-structure/2-staging.md (hunk 5 above)
Comment (patkearns10) [reply website/docs/guides/best-practices/how-we-structure/2-staging.md]: agreed they should sketch it out, but thought it doesnt have to be spreadsheets. I am not going to die on that hill though, so ill remove the suggestion!

**Review by gwenwindflower — APPROVED** [review]
love it, thanks for these! :shipit:

**Pull request merged**